.github/workflows/main.yml

```diff
@@ -0,0 +1,42 @@
+name: Build & Release
```

Comment:
This implies releasing to GitHub releases, which it doesn't, a more fitting name is needed